### PR TITLE
Update 7e45a7e6ab3afcf99120e97aedf84e706e43d829ddc610ad667a85a3.md

### PR DESCRIPTION
**Pull request by pgwadapool**
# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have checked my pool's registration status using https://pool.vet and have passed all checks.
- [ ] I have create an <IDENTITY_ID>.md for every stake pool operator of my pool file inside https://github.com/armada-alliance/armada-alliance/tree/main/services/website/content/en/stake-pools
- [ ] I have create a <POOL_ID>.md file for my pool inside https://github.com/armada-alliance/armada-alliance/tree/main/services/website/content/en/identities
- [ ] I have included my pool's ID in my <POOL_TICKER>.json file
- [ ] I have included my personal telegram handle in my <IDENTITY_ID>.md file

### CODE REVIEW

File: services/website/content/en/stake-pools/7e45a7e6ab3afcf99120e97aedf84e706e43d829ddc610ad667a85a3.md [hunk 1]
```diff
@@ -5,9 +5,27 @@ memberSince: 2021-06-15
...
+    deviceId: raspberry-pi-4-8gb
+    storage: WD_Black NVMe SSD 512GB
+  - role: relay 
+    deviceId: raspberry-pi-4-8gb
+    storage: WD_Black NVMe SSD 512GB
+  - role: relay
+    name: Backup1 
+    deviceId: raspberry-pi-4-8gb
+    storage: WD_Black NVMe SSD 512GB
+  - role: relay
```
Comment (AstroWa3l) [inline services/website/content/en/stake-pools/7e45a7e6ab3afcf99120e97aedf84e706e43d829ddc610ad667a85a3.md:22]: don't include the backups

**Review by AstroWa3l — COMMENTED** [review]
I would remove the backups, we don't care about the backups, just your "stake pool nodes that are running"

File: services/website/content/en/stake-pools/7e45a7e6ab3afcf99120e97aedf84e706e43d829ddc610ad667a85a3.md (hunk 1 above)
Comment (sublayerio) [reply services/website/content/en/stake-pools/7e45a7e6ab3afcf99120e97aedf84e706e43d829ddc610ad667a85a3.md:22]: backups are ok, just add `isBackup: true`

Comment (AstroWa3l) [reply services/website/content/en/stake-pools/7e45a7e6ab3afcf99120e97aedf84e706e43d829ddc610ad667a85a3.md:22]: but why? I don't find any value in knowing you have backups

Comment (AstroWa3l) [reply services/website/content/en/stake-pools/7e45a7e6ab3afcf99120e97aedf84e706e43d829ddc610ad667a85a3.md]: Well, if Sub says to do it, then do it :)

Comment (sublayerio) [reply services/website/content/en/stake-pools/7e45a7e6ab3afcf99120e97aedf84e706e43d829ddc610ad667a85a3.md]: it's alright, we can always scrub some data off. harder than to collect it.

**Comment by sublayerio** [comment]
Now it's looking great :)

**Pull request merged**